snapshot/pkg/controller/reconciler/reconciler.go

```diff
@@ -130,7 +127,7 @@ func (rc *reconciler) reconcile() {
 			rc.snapshotter.CreateVolumeSnapshot(snapshot)
 		} else {
 			// The VolumeSnapshot is in both states of world: check its OK and fix it eventually.
-			aswSnapshot := rc.actualStateOfWorld.GetSnapshot(name)
```

Comment:
log it here with info about snapshot data and timestamp

Reply:
I actually removed the whole else part. If snapshot is already in actual state, there is nothing to do. We make sure the snapshot is already ready before updating it into the actual state.